quickcomm/models.py

```diff
@@ -1,3 +1,125 @@
+import uuid
 from django.db import models
+from django.contrib.auth.models import User
+from django.core.validators import URLValidator
+
+# NOTE: The models in this file do not take into account how the site will
+# interact with other APIs. Ideally, we should be able to reuse the model
+# classes, but this logic is to be implemented later.
+
+# Type fields are left out as they can be added later and are redundant. Host
+# fields are kept as the if these classes are used to represent remote authors,
```

Comment:
I think this comment is meant to be 'fields are kept as if these classes are used...' Just a small typo with 'the' in the middle